src/gc/env/gcenv.base.h

```diff
@@ -281,6 +283,62 @@ inline uint8_t BitScanForward64(uint32_t *bitIndex, uint64_t mask)
 #endif // _MSC_VER
 }
 
+// Cross-platform wrapper for the _BitScanReverse compiler intrinsic.
+inline uint8_t BitScanReverse(uint32_t *bitIndex, uint32_t mask)
+{
+#ifdef _MSC_VER
+    return _BitScanReverse((unsigned long*)bitIndex, mask);
+#else // _MSC_VER
+    unsigned char ret = FALSE;
```

Comment:
clang/gcc have `__builtin_clz`, we actually had a wrapper in `pal.h` until about 10 days ago, when I removed it (because I removed the only usage we could find): <https://github.com/dotnet/coreclr/commit/cee8cc77e91f5a2c249da29b95f3ee0d10bbed7a#diff-b836af8f59cbb2bd02de8224c57d0002>

Reply:
nevermind, I put this on the wrong function and missed the line that was using the right intrinsic